corehq/apps/change_feed/document_types.py

```diff
@@ -2,64 +2,40 @@
 from __future__ import unicode_literals
 from collections import namedtuple
 
-from corehq.apps.app_manager.util import app_doc_types
 from corehq.apps.change_feed.exceptions import MissingMetaInformationError
 from couchforms.models import all_known_formlike_doc_types
 from dimagi.utils.couch.undo import DELETED_SUFFIX
 from pillowtop.feed.interface import ChangeMeta
 
-CASE = 'case'
-FORM = 'form'
-DOMAIN = 'domain'
-META = 'meta'
-COMMCARE_USER = 'commcare-user'
-WEB_USER = 'web-user'
-GROUP = 'group'
-APP = 'app'
+GROUP_DOC_TYPES = ('Group', 'Group-Deleted')
+WEB_USER_DOC_TYPES = ('WebUser', 'WebUser-Deleted')
+MOBILE_USER_DOC_TYPES = ('CommCareUser', 'CommCareUser-Deleted')
+DOMAIN_DOC_TYPES = ('Domain', 'Domain-Deleted', 'Domain-DUPLICATE')
+CASE_DOC_TYPES = ('CommCareCase', 'CommCareCase-Deleted')
+SYNCLOG_DOC_TYPES = ('SyncLog', 'SimplifiedSyncLog')
 
 
 DocumentMetadata = namedtuple(
-    'DocumentMetadata', ['raw_doc_type', 'primary_type', 'subtype', 'domain', 'is_deletion']
```

Comment:
removed this as it wasn't really used anywhere and just added a layer of indirection